### DISCUSSION TITLE
General MOOSE capabilities

### DISCUSSION THREAD

**Discussion by POLonge** [idaholab/moose#25634] [Q&A Modules: General]
Hi everyone,

Iam trying to develop diverse coupled Multiphysics models for COR-EOR and CO2 storage processes using different MOOSE module. Please I need assistance in ticking Yes or No on the capabilities that applies to MOOSE.

![image](https://github.com/idaholab/moose/assets/146491923/45cea460-be52-4676-b15a-8dc230baf36c)
![image](https://github.com/idaholab/moose/assets/146491923/1e70e0f5-76ad-424b-b0d4-bb5188927822)
![image](https://github.com/idaholab/moose/assets/146491923/57275211-6c30-4aed-8635-6524db74183f)
![image](https://github.com/idaholab/moose/assets/146491923/f87e8dc8-7d8f-4428-8c1a-4041b27ba5aa)

**Comment by GiudGiud** [comment]
Hello

Do you mind creating a Google doc or something easier to work with ?

Guillaume

**Comment by POLonge** [comment]
Hi GuidGiud,
Thanks for help, I have kept a google doc link. 

https://docs.google.com/document/d/11HaMZWq__qiRubsbD-eGIg1vRlGpwgyWBa5Mvp2t5BQ/edit?usp=sharing

**Reply by GiudGiud** [reply]
@1runer @rpodgorney could you please help out here

**Reply by GiudGiud** [reply]
@cpgr too please

**Reply by cpgr** [reply]
I can't edit the document, but I think you can put a yes for all fluid flow; yes for all transport except electrochemical and colloids; yes for the geochemistry except for Davies and Pitzer and the isotope ones; yes for the microbial (will have to double check but I think this is correct); yes for coupling except electrical double layer; yes for operator splitting and global implicit, FE and FV for spatial discretisation, heaps of options for time discretisation.

Not sure what you mean exactly by high peclet transport - do you mean some sort of stabilisation? Also not sure about elasto-plastic cap model so probably a no for that one.

**Reply by GiudGiud** [reply]
@POLonge please make sure edit rights are pretty loose so the community can pitch in

Thank you

**Reply by POLonge** [reply]
@GiudGiud @cpgr @rpodgorney 
This link is editable now. Thanks
https://docs.google.com/document/d/11HaMZWq__qiRubsbD-eGIg1vRlGpwgyWBa5Mvp2t5BQ/edit?usp=sharing

**Reply by GiudGiud** [reply]
@1runer or @rpodgorney  could you please fill the geomechanics section?

**Comment by rpodgorney** [comment]
Zscaler blocks access



Robert Podgorney, Ph.D.
Fellow and Senior Advisor   |   Advanced Scientific Computing
Idaho National Laboratory   |   1955 Fremont Ave.   |   Idaho Falls, ID   |   83415
***@***.******@***.***>   |   208-526-1524
_______________________________

[signature_4227069542]

Statement of Confidentiality
The contents of this e-mail message and any attachments are confidential and are intended solely for addressee. The information may also be legally privileged. This transmission is sent in trust, for the sole purpose of delivery to the intended recipient. If you have received this transmission in error, any use, reproduction or dissemination of this transmission is strictly prohibited. If you are not the intended recipient, please immediately notify the sender by reply e-mail or phone and delete this message and its attachments, if any.


From: POLonge ***@***.***>
Date: Wednesday, October 11, 2023 at 9:31 AM
To: idaholab/moose ***@***.***>
Cc: Robert K. Podgorney ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL] Re: [idaholab/moose] General MOOSE capabilities (Discussion #25634)

@GiudGiud<https://github.com/GiudGiud>
This link is editable now. Thanks
https://docs.google.com/document/d/11HaMZWq__qiRubsbD-eGIg1vRlGpwgyWBa5Mvp2t5BQ/edit?usp=sharing
